### ISSUE TITLE
Dropbox won't auto-start

### ISSUE THREAD

**Issue by Gabriel-p** [nathandyer/elementary-dropbox-mods#11]
I used this script to install Dropbox and bypass [this issue](https://bugs.launchpad.net/elementaryos/+bug/1452040) but now Dropbox won't auto-sytart with the system.

If I try:

    dropbox status

I get an error saying Dropbox isn't installed. I'm not sure why this happens since Dropbox is definitely installed?

Adding:

    ~/.dropbox-dist/dropboxd start &

to the start-up applications in `Preferences` does not work. Each time the system is loaded, I have to manually start Dropbox using the command above.

**Comment by guillermochica** [comment]
I'm not sure why, but dropbox status tells me too that dropbox isn't installed. However, it is installed because it works (I can upload files, I receive notifications from shared folders...). 

To auto-start dropbox, what I did was:

-Create a file called Dropbox.desktop in /home/youruser/.config/autostart with the following inside:
`[Desktop Entry]
Name=Dropbox
GenericName=File Synchronizer
Comment=Sync your files across computers and to the web
Exec=dropbox start -i
Terminal=false
Type=Application
Icon=dropbox
Categories=Network;FileTransfer;
StartupNotify=false
Exec=/home/guillermo/.dropbox-dist/dropboxd
X-GNOME-Autostart-enabled=true`

That way it should appear in System configuration -> Applications -> Auto start (Maybe it has other name, but I don't know because my system is in Spanish but I think you get the idea).

**Comment by Gabriel-p** [comment]
Thanks for the answer Guillermo, I'll try it as soon as I get to my laptop at work.

BTW, my system is in Spanish too (I'm from Argentina :)

Gracias!

**Comment by Gabriel-p** [comment]
Nope, not only it did not work, now the dropbox icon won't show up in Wingpanel after starting it manually via terminal (ie: ~/.dropbox-dist/dropboxd start &).

**Comment by flatcap-monty** [comment]
It looks like dropbox was starting anyway, as changes I made were registering on the web interface, but the icon was still missing.
I had a bit of a play around, and the icon returned after running the dropbox-elementary.sh script again, not in the terminal, literally just clicked the icon.

**Comment by Heart1010** [comment]
Dropbox updates itself in the background... After such an update do we lose the icon again and we have to run the install script again?

**Comment by Gabriel-p** [comment]
I get the same issue with `dropbox status` using [this other script](https://github.com/zant95/elementary-dropbox).

Opened a report over there too: https://github.com/zant95/elementary-dropbox/issues/4

**Comment by Gabriel-p** [comment]
The script I mention above fixes the no auto-start-up issue.

The issue where `dropbox status` returns the *Dropbox is not installed* error is also present when using that script though.
